### DISCUSSION TITLE
Is it possible to search for folders instead of individual videos?

### DISCUSSION THREAD

**Discussion by wsdookadr** [jellyfin/jellyfin#12535] [Q&A]
Is it possible to search for folders instead of individual videos?
Alternatively, is it possible to click on something for a video and reach the folder that video belongs to?

Thanks!

**Comment by TheMelmacian** [comment]
In the server settings -> libraries/Display you can activate "folder view".
This adds a separate library where your media is sorted into folders representing the real folder structure of your media libraries.

The "media information" (three dots on the media item) shows the complete path to the media file.
